src/components/canrawview/canrawview_p.h

```diff
@@ -29,17 +29,20 @@ class CanRawViewPrivate : public QWidget {
     {
         ui->setupUi(this);
 
-        tvModel.setHorizontalHeaderLabels({ "rowID", "time", "id", "dir", "dlc", "data" });
+        tvModel.setHorizontalHeaderLabels({ "rowID", "timeDouble", "time", "idInt", "id", "dir", "dlc", "data" });
         ui->tv->setModel(&tvModel);
         ui->tv->horizontalHeader()->setSectionsMovable(true);
-        //ui->tv->setColumnHidden(0, true);
+        ui->tv->horizontalHeader()->setSortIndicator(0, Qt::AscendingOrder);
+        ui->tv->setColumnHidden(0, true);
+        ui->tv->setColumnHidden(1, true);
+        ui->tv->setColumnHidden(3, true);
 
         connect(ui->pbClear, &QPushButton::pressed, this, &CanRawViewPrivate::clear);
         connect(ui->pbDockUndock, &QPushButton::pressed, this, &CanRawViewPrivate::dockUndock);
 
-        connect(ui->tv->horizontalHeader(), &QHeaderView::sectionClicked, [=]( const int &logicalIndex ) { sort( logicalIndex, clickedIndexes ); });
-        connect(&tvModel, &QAbstractItemModel::rowsInserted, [=]() { update( clickedIndexes ); });
-
+        connect(ui->tv->horizontalHeader(), &QHeaderView::sectionClicked,
+            [=](const int& logicalIndex) { sort(logicalIndex); });
```

Comment:
I believe that passing int as const & makes no sense . const & makes sense when you want to avoid copying big objects.